aiohttp/streams.py

```diff
@@ -265,7 +274,7 @@ def end_http_chunk_receiving(self):
                 if ichar:
                     not_enough = False
 
-                if line_size > self._limit:
+                if line_size > self._high_water:
```

Comment:
Previously, max line size was `2 ** 16` by default, but not it's `2 ** 17`. Is that intentional?

Reply:
Yes, it is intentional.
Line size should be less than high watermark is my mind -- was larger than low watermark.

Reply:
Ok.